File: Week 1/Mario.c
#include <stdio.h>
#include <cs50.h>
int main(void)
{
	int height;
	do
	{
		height = get_int("Please, type a height length between [1,8]: ");
	}while(height < 1 || height > 8);
	int row,column;
	for(row=0;row<height;row++)
	{
		for(column=0;column<height;column++)
			if(row+column>= height-1)
				printf("#");
			else
				printf(" ");
		printf("\n");
	}
	return 0;
}

File: Week 6/mario.py
from cs50 import get_int

height = 0
while(True):
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for row in range(height):
    for col in range(height):
        if col + row < height - 1:
            print(" ", end="")
        else:
            print("#", end="") 
    print() 

for row in range(height):
    for col in range(height):
        if col <= row:
            print("#", end="") 
    print() 
 
